frontend/server/src/DAO/QualityNominations.php

```diff
@@ -366,7 +366,7 @@ public static function getNominations(
         }
 
         // TODO(#3696): Change to ASC once duplicates are removed.
-        $sqlOrder = ' ORDER BY p.alias ASC, qn.qualitynomination_id DESC';
+        $sqlOrder = ' ORDER BY qn.time ASC, p.alias ASC, qn.qualitynomination_id DESC';
```

Comment:
creo que ya podemos quitar el TODO de arriba, que dice que `p.alias ASC` es un workaround temporal: entonces en vez de agregarle un workaround para el workaround, podemos eliminar ambos de una vez!

```suggestion
        $sqlOrder = ' ORDER BY qn.qualitynomination_id ASC';
```